packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -2858,6 +2881,70 @@ export class LitNodeClientNodeJs
     return signatures;
   };
 
+  /**
+   * Retrieves the PKP session signatures.
+   *
+   * @param params - The parameters for retrieving the PKP session signatures.
+   * @returns A promise that resolves to the PKP session signatures.
+   * @throws An error if any required parameter is missing or if both `litActionCode` and `ipfsId` are provided.
+   */
+  getPkpSessionSigs = async (
+    params: GetPkpSessionSigs
+  ): Promise<SessionSigsMap> => {
+    const chain = params?.chain || 'ethereum';
+
+    const pkpSessionSigs = this.getSessionSigs({
```

Comment:
If we're using this internally we should use a separate public function for getting EOA sessionSigs as it can still be used to get PKP sessionSigs?

Reply:
yes we can do that, this is an old/untouched code. We can have a separate PR to address & clean up all these

Reply:
We can do this later